veps/sig-network/ns-admin-nichotplug.md

```diff
@@ -0,0 +1,212 @@
+# VEP #29: NIC Hotplug / Hotunplug For Namespace Admins Following VMIM Hardening
```

Comment:
> Following VMIM Hardening

I suggest the name will focus on what it does, e.g. integration with rollout strategy mechanism.